tests/specs/graph/jsr/FastCheck_Functions.txt

```diff
@@ -192,9 +192,9 @@ Fast check https://jsr.io/@scope/a/1.0.0/mod.ts:
   export function test6<T = PublicOther>(param?: number): T;
   export function test7(param?: number): number;
   export function test7(param?: number, param2?: PublicOther2): number;
-  function test8(param: number): number;
-  function test8(param: string): string;
-  function test8(param0?: any): any;
+  declare function test8(param: number): number;
+  declare function test8(param: string): string;
+  declare function test8(param0?: any): any;
```

Comment:
Main fix is here
![image](https://github.com/denoland/deno_graph/assets/1609021/4bda795b-1abd-4694-9351-89d0e3d0fd55)